tutorials/nservicebus-101/index.md

```diff
@@ -4,32 +4,19 @@ suppressRelated: true
 reviewed: 2017-01-26
 ---
 
-In this course, you will learn how to get working quickly with NServiceBus.
+partial: nsb101-intro-paragraph
 
+ * In [Lesson 1: Getting started](lesson-1/), which should take no longer than 10-15 minutes, you will learn how to set up your development environment and create your very first messaging endpoint.
```

Comment:
I know we're going for friendlier language I think we should still try to be more economical. For example "which should take no longer than x minutes" could be shortened to `(x minutes)` at the end of each summary.